ztests/suite/zqd/services.sh

```diff
@@ -6,7 +6,11 @@ function awaitfile {
   until [ -f $file ]; do
     let i+=1
     if [ $i -gt 5 ]; then
-      echo "timed out waiting for file \"$file\" to appear"
+      echo "timed out waiting for file \"$file\" to appear\n"
+      echo "minio log:\n"
+      cat minio.log
+      echo "zqd log:\n"
+      cat zqd.log
```

Comment:
The echo command already emits one newline. Are you sure you want to add another one inside these strings?

Reply:
whoops